images/pyspark-notebook/Dockerfile

```diff
@@ -34,20 +34,18 @@ ARG scala_version
 # But it seems to be slower, that's why we use the recommended site for download
 ARG spark_download_url="https://dlcdn.apache.org/spark/"
 
-# Configure Spark
-ENV SPARK_VERSION="${spark_version}" \
-    HADOOP_VERSION="${hadoop_version}" \
-    SCALA_VERSION="${scala_version}" \
-    SPARK_DOWNLOAD_URL="${spark_download_url}"
```

Comment:
These env variable names are not something common, it's something we made up, so I don't think we should set them in the images.